### PR TITLE
DX: Add dev-tools directory to gitattributes as export-ignore

### PR DESCRIPTION
**Pull request by alexmanno**

### CODE REVIEW

File: .gitattributes [hunk 1]
```diff
@@ -10,6 +10,7 @@ tests/Test/AbstractTransformerTestCase.php export-ignore
 
 .appveyor.yml export-ignore
 .circleci/ export-ignore
+dev-tools/ export-ignore
```
Comment (dmvdbrugge) [inline .gitattributes]: I'm neither for nor against this, but please do put it in the correct place alphabetically speaking (after `check_trailing_spaces.sh`).

**Comment by keradus** [comment]
@SpacePossum , anything against going 2.12 ?

**Comment by SpacePossum** [comment]
2.12 sounds great to me as well :)

**Comment by SpacePossum** [comment]
@alexmanno can you retarget the PR to 2.12? if you're not familiar with it or don't have the time I'm happy to do it on your behalf.

**Comment by alexmanno** [comment]
@SpacePossum I'm not very confident with this, can you do it? Many thanks!

**Comment by SpacePossum** [comment]
sure thing, done :)  please note that I force pushed to your branch to change the head

**Comment by keradus** [comment]
Thank you @alexmanno.

**Pull request merged**